beacon_node/beacon_processor/src/metrics.rs

```diff
@@ -49,6 +49,14 @@ pub static BEACON_PROCESSOR_WORKERS_ACTIVE_TOTAL: LazyLock<Result<IntGauge>> =
             "Count of active workers in the gossip processing pool.",
         )
     });
+pub static BEACON_PROCESSOR_WORKERS_ACTIVE_GAUGE_BY_TYPE: LazyLock<Result<IntGaugeVec>> =
+    LazyLock::new(|| {
+        try_create_int_gauge_vec(
+            "beacon_processor_workers_active_gauge_by_type",
+            "Int gauge of the number of active workers per work type",
+            &["type"],
+        )
+    });
```

Comment:
Do you think we should remove the metric without the worker type? We can always re-derive it by summing across the `type`.